muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/WorkspaceHelpPageBackingBean.java

```diff
@@ -60,6 +65,11 @@ public String init() {
     if (workspaceEntity == null) {
       return NavigationRules.NOT_FOUND;
     }
+    
+    if (!sessionController.hasWorkspacePermission(MuikkuPermissions.MANAGE_WORKSPACE_HELP, workspaceEntity)) {
```

Comment:
This backing bean is responsible of showing workspace help instead of managing it, so this permission check is invalid.